project.clj

```diff
@@ -9,11 +9,11 @@
   :jvm-opts ["-server"]
   :java-source-paths ["src"]
   :plugins [[lein-bin "0.3.5"]
-            [jonase/eastwood "0.2.6"]
-            [lein-cloverage "1.0.11"]
-            [lein-cljfmt "0.5.7"]]
+            [lein-cloverage "1.0.11" :exclusions [org.clojure/clojure]]
```

Comment:
The exclusion commit header is a bit vague -- would you please update to specify its purely a org.clojure/clojure exclusion update? If there is a future exclusion related problem, it will make searching the commit history easier (I can't imagine excluding org.clojure/clojure being of interest for an exclusion related issue)

Reply:
will add comment (it was suggested by the compiler).